Doc/whatsnew/3.7.rst

```diff
@@ -231,8 +241,17 @@ The forced UTF-8 mode can be used to change the text handling behavior in
 an embedded Python interpreter without changing the locale settings of
 an embedding application.
 
-The UTF-8 mode is enabled by default when the locale is "C".  See
-:ref:`whatsnew37-pep538` for details.
+While :pep:`540`'s UTF-8 mode has the benefit of working regardless of which
+locales are available on the running system, it has the downside of having no
+effect on extension modules, child processes running non-Python applications,
```

Comment:
If you want to enumerate things not affected by the utf8 mode, I would also suggest to mention C libraries such as readline.